kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -175,9 +177,13 @@ oriented data synchronization.
         });
       },
       checkAnswer() {
-        const answer = this.$refs.contentRenderer.checkAnswer();
-        if (answer) {
-          this.answerGiven(answer);
+        if (!this.checkingAnswer) {
+          this.checkingAnswer = true;
+          const answer = this.$refs.contentRenderer.checkAnswer();
+          if (answer) {
+            this.answerGiven(answer);
+          }
+          this.checkingAnswer = false;
```

Comment:
As I think more about this, it's hard for me to imagine a case where the function would be re-entered with `checkingAnswer == true` because there are no promises or callbacks involved...